pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -199,10 +199,12 @@ func (cb *ClusterBuilder) buildDefaultCluster(name string, discoveryType cluster
 
 	switch discoveryType {
 	case cluster.Cluster_STRICT_DNS:
-		c.DnsLookupFamily = cluster.Cluster_V4_ONLY
-		dnsRate := gogo.DurationToProtoDuration(cb.push.Mesh.DnsRefreshRate)
-		c.DnsRefreshRate = dnsRate
-		c.RespectDnsTtl = true
+		if cb.push.Mesh.DnsRefreshRate != nil {
```

Comment:
If respect_dns_ttl is enabled, DNS record TTLs and dns_refresh_rate are used to control DNS refresh rate. For strict DNS cluster, if the minimum of all record TTLs is 0, dns_refresh_rate will be used as the cluster’s DNS refresh rate. Here is the relavant doc on this. So I removed the mesh config DnsRefreshRate default i.e. 5s which is also equivalent to Envoy default.